### PR TITLE
Move stage vars into dedicated subdirectory

### PR DESCRIPTION
**Pull request by stvstnfrd**
See: Stanford-Online/edx-secret#164

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Some subset of @Stanford-Online/edx-secret

**Comment by caesar2164** [comment]
NICE, :+1:

**Comment by stvstnfrd** [comment]
@caesar2164 I just pushed (and squashed, sorry) a few more commits to make these same changes for all of the stage machines. Please take another look through.

File: playbooks/edx-west/stage-app.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@
     not_prod: true
   vars_files:
     - "{{ secure_dir }}/vars/common/edxapp.yml"
-    - "{{ secure_dir }}/vars/edxapp_{{ env }}_vars.yml"
-    - "{{ secure_dir }}/vars/edxapp_{{ env }}_users.yml"
-    - "{{ secure_dir }}/vars/shib_stage_vars.yml"
+    - "{{ secure_dir }}/vars/{{ env }}/edxapp.yml"
+    - "{{ secure_dir }}/vars/{{ env }}/users.yml"
+    - "{{ secure_dir }}/vars/stage/shib.yml"
```
Comment (caesar2164) [inline playbooks/edx-west/stage-app.yml]: why not `- "{{ secure_dir }}/vars/{{ env }}/shib.yml"` ?

**Comment by caesar2164** [comment]
Looks pretty good, but could we extend the use of the `{{ env }}` pattern to all the files?

**Comment by stvstnfrd** [comment]
w/r/t the `{{ env }}` pattern:

We currently use a mix of the forms. I'm slowly transitioning to the
new, parameterized version, but I think it's only in place for the app
and worker plays.

We don't want to add this convention to the files not yet using it (not
yet, at least; it requires additional changes elsewhere), but we can go
ahead and add it in the files already supporting the convention.

Good catch.

**Comment by stvstnfrd** [comment]
@caesar2164 Change made; turned out to only be in place for `stage-app.yml`. Will squash before merging.

**Comment by caesar2164** [comment]
cool, :+1: when tests pass and squashed!

**Comment by kluo** [comment]
+1

**Pull request closed without merging**